typescript/packages/plugins/coingecko/src/parameters.ts

```diff
@@ -0,0 +1,37 @@
+import { z } from "zod";
+
+export const getTrendingCoinsParametersSchema = z.object({
+	limit: z.number().optional().describe("The number of trending coins to return. Defaults to all coins."),
+	include_platform: z.boolean().optional().describe("Include platform contract addresses (e.g., ETH, BSC) in response"),
+});
+
+export const getCoinPriceParametersSchema = z.object({
+	coinId: z.string().describe("The ID of the coin on CoinGecko (e.g., 'bitcoin', 'ethereum')"),
+	vsCurrency: z.string()
+		.default("usd")
+		.describe("The target currency to get price in (e.g., 'usd', 'eur', 'jpy')"),
+	include_market_cap: z.boolean()
```

Comment:
These parameters are not being used. Do we need them? Also I would keep them consistent making them camel case